### PR TITLE
Preprod: movie: Lucy (2014) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
This went from "looks promising" to "wth" pretty quickly. Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2010s.st.txt [hunk 1]
```diff
@@ -1645,6 +1645,58 @@ human nature [It was human nature to romanticize war, according to the malevolen
...
+https://en.wikipedia.org/wiki/Lucy_(2014_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+organized crime [The ruthless Mr. Jang was the mastermind behind an international drug smuggling ring. The story follows Lucy as she becomes forcibly made to serve as a drug mule for Mr. Jang and his fellow criminals.]
+speculative ability [The story follows Lucy as she acquires a host of superpowers after an experimental, illicit drugs gets into her bloodstream. Her superpowers included, super intelligence, mind reading, telekinesis, an ability to manipulate radio and other EM waves, and even time travel.]
+human nature [A message of the film is that a loss of humanity comes with increased intelligence.]
+apotheosis [The story culminated with Lucy transcending space and time upon reaching 100% usage of her brain capacity, and becoming one with the universe or some such thing.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2010s.st.txt:1675]: A slight broadening of our definition, but I guess it's fair. Might update our theme definition to fit.

File: notes/stories/film/film-scifi-2010s.st.txt [hunk 2]
```diff
@@ -1645,6 +1645,58 @@ human nature [It was human nature to romanticize war, according to the malevolen
...
+chemicals [Exposure to the CPH4 synthesized chemical endowed Lucy with heightened physical and mental abilities.]
+adapting to life in a foreign culture [Lucy was studying in Taipei without knowing the language.]
+law enforcement [The police captain Pierre Del Rio led a search for a trio of drug mules.]
+what it is like on a passenger flight [Lucy made something of a scene on an otherwise routine passenger flight to Paris.]
+telekinetic ability [Lucy acquired this power which she used to, among other things, make the bullets fall out of a gun that was being pointed at her.]
+shapeshifting [Lucy acquired the ability to transform her body at will. For instance, she changed the color of her hair and later changed her hand into a completely different hand.]
+time travel [Lucy ultimately acquired an innate ability to travel in time, which she used to go back to see the dinosaurs and the Big Bang itself.]
+
+:: Other Keywords
+electromagnetic wave manipulation ability [Lucy acquired an ability to manipulate radio and other electromagnetic waves, enabling her to make herself appear on peoples computer screens at will.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2010s.st.txt:1697]: https://www.themeontology.org/theme.php?name=electricity%20manipulation%20ability ?

Comment (odinlake) [reply notes/stories/film/film-scifi-2010s.st.txt:1697]: I also though we had a theme about controlling machines in general but can't find it

**Review by odinlake — APPROVED** [review]
Comments inline

File: notes/stories/film/film-scifi-2010s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2010s.st.txt]: Erring on the side of caution here, I revised "apotheosis" to be the more general "what if I underwent a physical metamorphosis". I noticed we have some other cases of "what if I underwent a physical metamorphosis", like the case of John Doe in tng3x25.

**Pull request merged**